checkov/terraform/checks/graph_checks/aws/SGAttachedToResource.yaml

```diff
@@ -37,18 +37,19 @@ definition:
         - aws_launch_configuration
         - aws_launch_template
         - aws_lb
+        - aws_memorydb_cluster
+        - aws_mq_broker
```

Comment:
it looks like `aws_mq_broker` is twice there. thanks for reording, didn't see that before 😅

Reply:
Oh, indeed, let me remove it.